pages/resources/collectibles.mdx

```diff
@@ -0,0 +1,802 @@
+
+###### Shelf Shop Block Structure
+
+| Field            | Type             | Description                                                                                     |
+| ---------------- | ---------------- | ----------------------------------------------------------------------------------------------- |
+| category_sku_id? | snowflake        | The [collectible category](#collectible-category-structure) SKU ID                              |
+| name             | string           | Name of the shop block                                                                          |
+| ranked_sku_ids   | array[snowflake] | The SKU IDs ranked by popularity in the [collectible category](#collectible-category-structure) |
+| type             | integer          | The [shop block type](#shop-block-type)                                                         |
+
+###### Countdown Timer Shop Block Structure
+
+| Field      | Type    | Description                             |
+| ---------- | ------- | --------------------------------------- |
+| type       | integer | The [shop block type](#shop-block-type) |
+| title      | string  | The title of the countdown timer        |
+| body       | string  | The body text of the countdown timer    |
+| banner_url | string  | The URL of the countdown timer banner   |
+| end_time   | Date    | The end time of the countdown timer     |
+| text_color | string  | The hex color code of the text          |
+
+###### Immersive Banner Shop Block Structure
+
+| Field               | Type                                           | Description                                    |
+| ------------------- | ---------------------------------------------- | ---------------------------------------------- |
+| type                | integer                                        | The [shop block type](#shop-block-type)        |
+| title               | string                                         | The title of the immersive banner              |
+| body                | string                                         | The body text of the immersive banner          |
+| help_center_url     | string                                         | The URL to the help center article             |
+| text_color          | string                                         | The hex color code of the text                 |
+| end_time            | ?ISO8601 timestamp                             | The end time of the immersive banner           |
+| banner_asset        | [asset config](#asset-config-structure) object | The banner asset                               |
+| banner_url          | string                                         | The URL of the immersive banner image          |
+| banner_animated_url | string                                         | The URL of the animated immersive banner image |
+
+### Collectibles Marketing Object
```

Comment:
object is only used once so it can just be a standalone structure under the endpoint